pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,55 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v1.0.38/quarto-1.0.38-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+  ];
+
+  installPhase = ''
+      mkdir -p $out/bin
+      mkdir -p $out/share
+
+      rm bin/tools/deno
+      rm bin/tools/pandoc
+      rm bin/tools/esbuild
+
+      mv bin/* $out/bin
+      mv share/* $out/share
+
+      ln -s ${deno.outPath}/bin/deno $out/bin/tools/deno
+      ln -s ${pandoc.outPath}/bin/pandoc $out/bin/tools/pandoc
+      ln -s ${esbuild.outPath}/bin/esbuild $out/bin/tools/esbuild
+  '';
+
+  meta = with lib; {
+    description = "Open-source scientific and technical publishing system built on Pandoc. ";
+    longDescription = ''
+        Quarto is an open-source scientific and technical publishing system built on Pandoc.
+        Quarto documents are authored using markdown, an easy to write plain text format.
+    '';
+    homepage = "https://quarto.org/";
+    changelog = "https://github.com/quarto-dev/quarto-cli/releases/tag/v${finalAttrs.version}";
+    license = licenses.gpl2;
```

Comment:
The license is GPLv2 or later, `gpl2Plus`.

Reply:
Solved in https://github.com/NixOS/nixpkgs/pull/186697/commits/75fb646bbac5108469e8481ccf18be44a3b2edb2